apps/hostd/contexts/metrics/index.tsx

```diff
@@ -395,13 +396,28 @@ function useMetricsMain() {
       data,
       stats,
       config: {
-        enabledGraph: ['successful', 'active', 'pending', 'rejected', 'failed'],
-        enabledTip: ['successful', 'active', 'pending', 'rejected', 'failed'],
+        enabledGraph: [
+          'active',
```

Comment:
@n8maninger with the new breakdown, do these still work stacked and with a total per period? I've simply added them all to the existing graph:
<img width="574" alt="Screenshot 2024-08-26 at 12 26 47 PM" src="https://github.com/user-attachments/assets/909f314d-a5cf-431e-b06e-5ad12410279b">